console/src/assets/i18n/de.json

```
@@ -112,6 +114,11 @@
             "DELETE_TITLE":"User löschen",
             "DELETE_DESCRIPTION":"Sie sind im Begriff einen Benutzer endgültig zu löschen. Wollen Sie dies wirklich tun?"
         },
+        "SENDEMAILDIALOG":{
+            "TITLE":"Email Benachrichtigung senden",
+            "DESCRIPTION":"Klicken Sie den untenstehenden Button and die aktuelle Emailadresse zu versenden oder ändern Sie die Emailadresse in dem Feld.",
```

Something is wrong with this message, should it be something like this:

```suggestion
            "DESCRIPTION":"Klicken Sie den untenstehenden Button um ein verifizierungs Email an die aktuelle Adresse zu versenden oder ändern Sie die Emailadresse in dem Feld.",
```